dbt_subprojects/dex/models/_projects/paraswap/base/paraswap_v5_base_trades.sql

```diff
@@ -14,6 +14,7 @@
 {% set trade_event_tables = [
     source('paraswap_base', 'AugustusSwapper_evt_BoughtV3')
     ,source('paraswap_base', 'AugustusSwapper_evt_SwappedV3')
+    ,source('paraswap_base', 'AugustusSwapper_evt_SwappedDirect')
```

Comment:
Please add the following source schema in `sources/paraswap/base/paraswap_base_sources.yml`:
```
      - name: AugustusSwapper_evt_SwappedDirect
        loaded_at_field: evt_block_time
```